Assets/Code/MinigameTanglesLair/LevelSelectorBehavior.cs

```diff
@@ -7,14 +7,23 @@
 
 public class LevelSelectorBehavior : MonoBehaviour
 {
+    // public GameData gameData;
     public GameObject buttonPrefab;
     public GameObject content;
     public GameBehavior gameBehavior;
     private LevelButtonBehavior[] buttons;
+
     // Start is called before the first frame update
+    // After clicking the "Play Button" on the TanglesLair Title Screen, Start is called
+
     void Start()
     {
+        Debug.Log("LevelSelectorBehavior.cs -> Start");
         buttons = new LevelButtonBehavior[CTConstants.N_LEVELS];
+
+        GameData.InitCircuitsSaveData();
```

Comment:
So `GameData` here is the `MinigameTanglesLair/GameData` right? In that case this could just be named `InitSaveData` or `InitTanglesLairSaveData`.